packages/runtime/src/useCases/common/validation/ValidatableStrings.ts

```diff
@@ -1,5 +1,5 @@
 /**
- * @pattern id1[A-Za-z0-9]{32,33}
+ * @pattern did:e:[a-zA-Z\.]*:dids:[0-9a-f]{22}
```

Comment:
`did:e:[A-Za-z0-9.-]+:dids:[0-9a-f]{22}`

The domain part should allow for dashes and digits (e.g. _did:e:preprod-0.enmeshed.eu_), and shouldn't be optional (right?). Also, inside a character set `[...]` the period character is interpreted literally, so you don't need to escape it.

Reply:
Good catch!